staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -290,6 +288,10 @@ func (s *sharedIndexInformer) AddEventHandlerWithResyncPeriod(handler ResourceEv
 	s.startedLock.Lock()
 	defer s.startedLock.Unlock()
 
+	if s.stopped {
+		return
```

Comment:
Do you think this would be worth a log message so people could at least see why their attempt to add an event handler didn't do anything?